### ISSUE TITLE
Add a option to allow unsupported cpus to use efly

### ISSUE THREAD

**Issue by MCNGamez** [cabaletta/baritone#4494]
## Describe your suggestion
Add a option to use elytra fly with unsupported cpus.

## Settings
The command being used twice to overide the message about unsuported cpu.

## Context
This would allow a wider amount of people to use this feature

## Final checklist
- [x] I know how to properly use check boxes 
- [x] I have not used any OwO's or UwU's in this issue.

**Comment by ZacSharp** [comment]
Netherpathfinder cannot run on all systems. If you see that message it either doesn't recognize your system and doesn't know which executable to use, or it already tried and failed. (or you have a 32 bit system, it can't run on those)
The only true workaround would be a (low performance) pathfinder implementation in Java.

**Comment by Huber1** [comment]
Does the compatibility vary between different mod loaders?
On neoforge elytra tells me this:
```
Legacy architectures are not supported. Your CPU is amd64 and your operating system is Linux.
Supported architectures are 64 bit x86, and 64 bit ARM. Supported operating systems are Windows, Linux and Mac
```

On fablic this just works. Unfortunately I use some other mods on neoforge...
If relevant, i am using `baritone-standalone-neoforge-1.11.1-2-ga0f100f4.jar` from this workflow: https://github.com/cabaletta/baritone/actions/runs/12592616905

**Comment by ZacSharp** [comment]
No, that shouldn't happen. It's the same set of native libraries for all loaders.

**Comment by ZacSharp** [comment]
@Huber1 can you please open a new bugreport and add a log (there should be messages from nether-pathfinder in the log file).

**Comment by Huber1** [comment]
Planned that already. Will do it when i get home
